packages/origin-backend/src/utils/positiveBNStringValidator.ts

```diff
@@ -0,0 +1,14 @@
+import { ValidatorConstraint, ValidatorConstraintInterface } from 'class-validator';
+import { bigNumberify, BigNumber } from 'ethers/utils';
+
+@ValidatorConstraint()
+export class PositiveBNStringValidator implements ValidatorConstraintInterface {
```

Comment:
This should be a part of utils or other common package to exchange and backend

Reply:
It's not the same as the one in the exchange.